### ISSUE TITLE
Error "stop_reason": response_body["stop_reason"] when running 0.1.5

### ISSUE THREAD

**Issue by thekrol01** [langchain-ai/langchain-aws#56]
I have run this code example with langchain-aws v 0.1.4 and get no errors:
https://catalog.workshops.aws/building-with-amazon-bedrock/en-US/bonus/bedrock-templates

However, running the same code with: v0.1.5, I get the error: 
in: /site-packages/langchain_aws/llms/bedrock.py", line 260, in prepare_output
    "stop_reason": response_body["stop_reason"],
                   ~~~~~~~~~~~~~^^^^^^^^^^^^^^^
KeyError: 'stop_reason'

Uninstalling v0.15 and installing v0.14 resolved the error

**Comment by supreetkt** [comment]
I'm also seeing this error. Not seeing it with anthropic, but seeing it with many other models from AI21, Cohere, Amazon, etc.

**Comment by 3coins** [comment]
Looking into this. Seems related to #55

**Comment by 3coins** [comment]
Fixed in v0.1.6.
